applications/nkp-insights-management/1.7.1/helmrelease/helmrelease.yaml

```diff
@@ -7,7 +6,7 @@ metadata:
 spec:
   chartRef:
     kind: OCIRepository
-    name: ${releaseName}-${appVersion}-chart
+    name: nkp-insights-management-1.7.1
```

Comment:
@takirala 
is `${releaseName}` picked up from the folder name? i.e `nkp-insights-management` in this case.
and `${appVersion}` the subsequent folder, i.e `1.7.1` here?